samples/net/ws_echo_server/README.rst

```diff
@@ -0,0 +1,36 @@
+.. _websocket-server-sample:
+
+Websocket Server
+################
+
+Overview
+********
+
+The websocket-server sample application for Zephyr implements a websocket
+server. The websocket-server listens for incoming IPv4 or IPv6 HTTP(S)
+requests.
+
+The source code for this sample application can be found at:
+:file:`samples/net/ws_server`.
+
+Requirements
+************
+
+- :ref:`networking_with_qemu`
+
+Building and Running
+********************
+
+There are multiple ways to use this application. One of the most common
+usage scenario is to run websocket-server application inside QEMU. This is
+described in :ref:`networking_with_qemu`.
+
+Build websocket-server sample application like this:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/ws_echo_server
+   :board: qemu_x86
+   :goals: run
+   :compact:
+
+The default make BOARD configuration for this sample is ``qemu_x86``.
```

Comment:
It appears you interact with this sample app with a web browser, but you don't include that in these instructions.